op-service/sources/reth_db_test.go

```diff
@@ -0,0 +1,38 @@
+//go:build rethdb
+
+package sources
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRethDBReceiptsLoad(t *testing.T) {
+	t.Parallel()
+
+	// Goerli block #9942861, with only the first transaction persisted to the DB
```

Comment:
I wasn't sure why the block hash didn't match the canonical 9942861 until I read this. tbc, the other transactions aren't included in the block either. There's a tiny sepolia block @ https://sepolia.etherscan.io/block/4595459 you could use if you're worried that the fixture might not fit in github. Though this is also fine, albeit a bit surprising.

Reply:
I have a strong preference to using full valid block